src/vmm/src/linux/vstate.rs

```diff
@@ -1099,6 +1240,16 @@ impl Vcpu {
                     self.io_bus.write(0, u64::from(addr), data);
                     Ok(VcpuEmulation::Handled)
                 }
+                VcpuExit::MemoryFault { gpa, size, flags } => {
+                    let private = (flags & (1 << 3)/* KVM_EXIT_MEMORY_FAULT */) != 0;
```

Comment:
```suggestion
                    let private = (flags & kvm_bindings::KVM_EXIT_MEMORY_FAULT) != 0;
```

Reply:
This was a typo, the actual flag I was referring to was `KVM_MEMORY_EXIT_FLAG_PRIVATE`, but your point still stands. Fixed in latest version.